model/process/registry.yaml

```diff
@@ -11,6 +11,14 @@ groups:
         brief: >
           Process identifier (PID).
         examples: [1234]
+      - id: process.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the process. On Linux based systems, can be set to
```

Comment:
nit: can or SHOULD ?

Reply:
I didn't write this originally so I'm not sure what the original intent of the author was, but I imagine SHOULD is probably correct.

Reply:
Made that change in [cd4c335](https://github.com/open-telemetry/semantic-conventions/pull/1737/commits/cd4c3357d8325b865b3221e9236b14169226834b)